.circleci/config.yml

```diff
@@ -728,7 +729,25 @@ jobs:
             - run:
                   name: Run tests
                   command: |
-                      mvn --fail-fast -s .gravitee.settings.xml test --no-transfer-progress -Dintegration-tests-modules -Dskip.validation=true -T 2C
+                      cd gravitee-apim-integration-tests
+                      # List all tests
+                      circleci tests glob "src/test/java/**/*Test.java" | sed -e 's#^src/test/java/\(.*\)\.java#\1#' | tr "/" "." > all-tests
```

Comment:
I fear extracting the test name will not be good enough to handle properly the tests from inner classes